adapter/src/components/ErrorBoundary.js

```diff
@@ -12,7 +13,7 @@ const replaceNewlinesWithBreaks = text =>
         .split('\n')
         .reduce((out, line, i) => [...out, line, <br key={i} />], [])
 
-export class FatalErrorBoundary extends Component {
+export class ErrorBoundary extends Component {
```

Comment:
FatalErrorBoundary is repurposed into a reusable error boundary with the option of making it fullscreen or not, and configuring what happens on clicking 'refresh'